osu.Game/Screens/Play/SongProgress.cs

```diff
@@ -89,10 +92,7 @@ public SongProgress()
                     Alpha = 0,
                     Anchor = Anchor.BottomLeft,
                     Origin =  Anchor.BottomLeft,
-                    SeekRequested = delegate (float position)
-                    {
-                        OnSeek?.Invoke(firstHitTime + position * (lastHitTime - firstHitTime));
-                    },
+                    OnSeek = position => this.OnSeek?.Invoke(position),

```

Comment:
Since the Action names are the same, I feel that it makes it clearer what the code is doing by keeping `this`.